src/app/study-sequence/study-sequence.component.scss

```diff
@@ -0,0 +1,43 @@
+.back-to-view{
+    border-style: dashed;
+    height: 44px;
+}
+
+.container{
+    display: grid;
+    //Makes a grid with 3 columns
+    grid-template-columns: auto auto auto;
+    height: calc(100vh - 100px);
+}
+
+.card-pool{
+    outline: 1px solid blue;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    text-align: center;
+}
+
+.sequence-builder{
+    outline: 1px dashed magenta;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    display: grid;
+    grid-template-rows: auto auto auto;
+}
+
+.sequence-select{
+    outline: 1px dashed green;
+    height: 72px;
+    text-align: center;
+}
+
+.cards-in-sequence{
+    outline: 1px dashed purple;
+    height: calc(100vh - 128px);
```

Comment:
This should probably be 100vh - 200px. The 200px is the navbar (56px this was changed in the project and incorrect in the Figma which is 1px off), and then the select sequence and check-show-answer are both 72px totaling 200. With this height, the check-show-answer is pushed off of the screen.